warden/group/handler.go

```diff
@@ -384,27 +363,58 @@ func (h *Handler) AddGroupMembers(w http.ResponseWriter, r *http.Request, ps htt
 //       403: genericError
 //       500: genericError
 func (h *Handler) RemoveGroupMembers(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	var ctx = r.Context()
 	var id = ps.ByName("id")
 
+	if err := h.checkRequest(r, fmt.Sprintf(h.PrefixResource(GroupResource), id), "members.remove"); err != nil {
+		h.H.WriteError(w, r, err)
+		return
+	}
+
 	var m membersRequest
 	if err := json.NewDecoder(r.Body).Decode(&m); err != nil {
 		h.H.WriteError(w, r, errors.WithStack(err))
 		return
 	}
 
-	if _, err := h.W.TokenAllowed(ctx, h.W.TokenFromRequest(r), &firewall.TokenAccessRequest{
-		Resource: fmt.Sprintf(h.PrefixResource(GroupResource), id),
-		Action:   "members.remove",
-	}, Scope); err != nil {
+	if err := h.Manager.RemoveGroupMembers(id, m.Members); err != nil {
 		h.H.WriteError(w, r, err)
 		return
 	}
 
-	if err := h.Manager.RemoveGroupMembers(id, m.Members); err != nil {
+	w.WriteHeader(http.StatusNoContent)
+}
+
+func (h *Handler) OverwriteGroupMembers(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
```

Comment:
I would probably rename it to "UpdateGroup"

Reply:
I think I'm going for `UpdateGroupMembers` because the http PUT url is `GroupsHandlerPath + "/:id/members"` so only one group is affected, not all of them